block.go

```diff
@@ -350,6 +353,89 @@ type Divider struct {
 	// empty
 }
 
+type EquationBlock struct {
+	BasicBlock
+	Equation Equation `json:"equation"`
+}
+
+type Equation struct {
+	Expression string `json:"expression"`
+}
+
+type BreadcrumbBlock struct {
+	BasicBlock
+	Breadcrumb Breadcrumb `json:"breadcrumb"`
+}
+
+type Breadcrumb struct {
+	// empty
+}
+
+type ColumnBlock struct {
+	BasicBlock
+	Column Column `json:"column"`
+}
+
+type Column struct {
+	// Children should at least have 1 block when appending.
+	Children []Block `json:"children"`
+}
+
+type ColumnListBlock struct {
+	BasicBlock
+	ColumnList ColumnList `json:"column_list"`
+}
+
+type ColumnList struct {
+	// Children can only contain column blocks
+	// Children should have at least 2 blocks when appending.
+	Children []Block `json:"children"`
+}
+
+// NOTE: will only be returned by the API. Cannot be created by the API.
+// https://developers.notion.com/reference/block#link-preview-blocks
+type LinkPreviewBlock struct {
+	BasicBlock
+	LinkPreview LinkPreview `json:"link_preview"`
+}
+
+type LinkPreview struct {
+	URL string `json:"url"`
+}
+
+type LinkToPageBlock struct {
+	BasicBlock
+	LinkToPage LinkToPage `json:"link_to_page"`
+}
+
+type LinkToPage struct {
+	Type   string `json:"type"`
+	PageID string `json:"page_id"`
+}
+
+type TemplateBlock struct {
+	BasicBlock
+	Template Template `json:"template"`
+}
+
+type Template struct {
+	Text     []RichText `json:"text"`
+	Children []Block    `json:"children,omitempty"`
+}
+
+type SyncedBlock struct {
+	BasicBlock
+	Synced Synced `json:"synced_block"`
```

Comment:
Let's rename propery to follow naming convention 
```suggestion
	SyncedBlock Synced `json:"synced_block"`
```

Reply:
Will do! Was wondering your opinion about the overall Block name. Technically the block is called a `synced_block` so, using the overall pattern would have it called `SyncedBlockBlock` which I found rather weird. I think that it does sound better as it is (the top block name), but I'd like to know what you think.